webservice/app/Http/Controllers/AuthController.php

```diff
@@ -93,6 +92,31 @@ public function sendLockoutResponse(Request $request)
         return $this->responseWithTooManyRequests($message);
     }
 
+    /**
+     * Revoke the user's token.
+     *
+     * @return \Illuminate\Http\Response
+     */
+    public function revokeToken()
+    {
+        Auth::guard('api')->logout();
+
+        return $this->responseWithNoContent();
+    }
+
+    /**
+     * Refresh the user's token.
+     *
+     * @param Request $request
+     * @return \Illuminate\Http\JsonResponse
+     */
+    public function refreshToken(Request $request)
+    {
+        $token = Auth::guard('api')->refresh();
+
+        return $this->response(compact('token'));
```

Comment:
Will be returned a 500 error by the jwt lib and the user will be redirected to login page. The token can be updated as many times as needed for 2 weeks (default setting).